src/content-loader/index.js

```diff
@@ -31,11 +31,15 @@ module.exports = function(input) {
   const defaultHead = query.defaultHead
   const parsed = frontMatterParser(input)
 
+  // Normalize all data to JSON compaitable types
+  // See https://github.com/MoOx/phenomic/issues/397
+  const parsedData = JSON.parse(JSON.stringify(parsed.data))
```

Comment:
Maybe we should not export as json on the client side, since that not just json.